installer/install_sim.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys, os, shutil, subprocess
+from subprocess import CalledProcessError
+import traceback as tb
+from helper import *
+
+sim_source_path = ''
+sim_etc_dir = '/etc/dolphin'
+sim_var_dir = '/var/lib/dolphin'
+conf_file = os.path.join(sim_etc_dir, 'dolphin.conf')
+proj_name = 'SIM'
+
+def _activate():
+    path_to_activate = os.getcwd() + '/' + proj_name + '/bin/activate'
```

Comment:
This seems to be unused function, still, path calculation to venv activate seems to be wrong

Reply:
Yes, unused as of now..will handle changes later..currently fixing the path